packages/user-interface/src/components/Task.tsx

```diff
@@ -36,10 +35,10 @@ const Task = ({ task }: Props) => {
       relativeDate
       labels={labels}
       dateTime={task.verloopdatum}
-      link={taskUrl}
+      link={taskUrl ?? ""}
```

Comment:
Kan deze leeg zijn? zou verwachten dat als hij taak soort url heeft dat de url ook beschikbaar is.

Reply:
Dat komt doordat wanneer taak soort url is, url nog steeds null kan zijn volgens de definities. Ik kan daar ook dit doen:
`  if (task.soort === TaakSoort.Url) return task.url?.uri;`

Reply:
Zojuist besproken: omdat we niet kunnen garanderen dat bij soort url task.url niet null/undefined is, laten we het zo.